### PR TITLE
package chef plugin for smart proxy (DEB)

### PR DESCRIPTION
**Pull request by mmoll**
I can't test this, as I don't have any Chef around, ping @ares

scratchbuilds:
- http://ci.theforeman.org/job/packaging_build_deb_dependency/622/
- http://ci.theforeman.org/job/packaging_build_deb_dependency/623/
- http://ci.theforeman.org/job/packaging_build_deb_proxy_plugin/22/

### CODE REVIEW

File: dependencies/precise/chef-api/changelog [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ruby-chef-api (0.5.0-1) unstable; urgency=low
+
+  * Initial release
+
+ -- Michael Moll <mmoll@mmoll.at>  Sun, 16 Nov 2014 02:12:44 +0100
```
Comment (domcleal) [inline dependencies/precise/chef-api/changelog]: 2015? :)

File: dependencies/precise/logify/control [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Standards-Version: 3.9.5
+Homepage: https://github.com/sethvargo/logify
+XS-Ruby-Versions: all
+
+Package: ruby-logify
+Architecture: all
+XB-Ruby-Versions: ${ruby:Versions}
+Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter, 
+Description: lightweight logging library for Ruby
+ Logify is an incredibly light-weight Ruby logger with a developer-friendly API and no dependencies. It is intentionally very opinionated and is optimized for speed. This combination makes it perfect for command line applications.
```
Comment (domcleal) [inline dependencies/precise/logify/control]: Is wrapping necessary in debs?

File: dependencies/precise/chef-api/control [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+Maintainer: Michael Moll <mmoll@mmoll.at>
+Build-Depends: debhelper, gem2deb
+Standards-Version: 3.9.5
+Homepage: https://github.com/sethvargo/chef-api
+XS-Ruby-Versions: all
+
+Package: ruby-chef-api
+Architecture: all
+XB-Ruby-Versions: ${ruby:Versions}
+Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter, ruby-logify, ruby-mime-types
```
Comment (domcleal) [inline dependencies/precise/chef-api/control]: logify is ~> 0.1

File: plugins/smart_proxy_chef/control [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+Maintainer: Michael Moll <mmoll@mmoll.at>
+Build-Depends: debhelper, gem2deb
+Standards-Version: 3.9.3
+Homepage: https://github.com/theforeman/smart_proxy_chef
+XS-Ruby-Versions: all
+
+Package: ruby-smart-proxy-chef
+Architecture: all
+XB-Ruby-Versions: ${ruby:Versions}
+Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter, foreman-proxy (>= 1.7.0~rc1), ruby-chef-api (>= 0.5.0), ruby-logify (>= 0.2.0)
```
Comment (domcleal) [inline plugins/smart_proxy_chef/control]: Depends directly just on ruby-chef-api

File: plugins/smart_proxy_chef/copyright [hunk 5]
```diff
@@ -0,0 +1,28 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: ruby-smart-proxy-chef
+Source: https://github.com/theforeman/smart_proxy_chef
+
+Files: *
+Copyright: 2012-2013 Foreman developers <foreman-dev@googlegroups.com>
+License: GPL-3.0+
+
+Files: debian/*
+Copyright: 2013 Foreman developers <foreman-dev@googlegroups.com>
```
Comment (domcleal) [inline plugins/smart_proxy_chef/copyright]: 2015

**Comment by domcleal** [comment]
Looks about right!

File: dependencies/precise/chef-api/changelog (hunk 1 above)
Comment (mmoll) [inline dependencies/precise/chef-api/changelog]: nope, that was in the pipe since then :)

File: dependencies/precise/logify/control (hunk 2 above)
Comment (mmoll) [inline dependencies/precise/logify/control]: This is intended, see
https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Description

File: dependencies/precise/chef-api/control (hunk 3 above)
Comment (mmoll) [inline dependencies/precise/chef-api/control]: done

File: plugins/smart_proxy_chef/control (hunk 4 above)
Comment (mmoll) [inline plugins/smart_proxy_chef/control]: done

File: plugins/smart_proxy_chef/copyright (hunk 5 above)
Comment (mmoll) [inline plugins/smart_proxy_chef/copyright]: done

**Comment by ares** [comment]
@mmoll sorry, I won't get to testing anytime soon, I'd suggest building it anyway

**Comment by domcleal** [comment]
Thanks @mmoll!  Merged to deb/develop and deb/1.7, I'll kick off builds now.

**Pull request merged**